components/brave_ads/core/internal/serving/new_tab_page_ad_serving.cc

```diff
@@ -78,7 +78,6 @@ void NewTabPageAdServing::GetEligibleAds(
 void NewTabPageAdServing::BuildUserModelCallback(
     MaybeServeNewTabPageAdCallback callback,
     const UserModelInfo& user_model) const {
-  CHECK(eligible_ads_);
```

Comment:
`CHECK` is not required because we call `IsSupported` within the flow of `MaybeServeAd`